Makefile

```diff
@@ -1006,6 +1021,11 @@ ubsan_crash_test_with_atomic_flush:
 	COMPILE_WITH_UBSAN=1 $(MAKE) crash_test_with_atomic_flush
 	$(MAKE) clean
 
+ubsan_crash_test_with_txn:
```

Comment:
Looks like tsan_crash_test_with_txn is missing?

Reply:
It is missing for atomic flush as well. Apparently it sets the tsan flag directly in the lego-determinator instead of relying it to be set from within Makefile: https://github.com/facebook/rocksdb/blob/master/build_tools/rocksdb-lego-determinator#L695